smrf/utils/utils.py

```diff
@@ -464,3 +464,36 @@ def grid_interpolate_deconstructed(tri, values, grid_points, method='linear'):
         return CloughTocher2DInterpolator(tri, values)(grid_points)
     elif method == 'linear':
         return LinearNDInterpolator(tri, values)(grid_points)
+
+def get_center(ds, mask_name=None):
```

Comment:
Will this be used anywhere else in SMRF? I see this a very specific to the `topo` class and it's probably better suited to live as a method in the `topo` class.

Reply:
I mentioned this a bit in the issue but originally I was thinking it would be useful for other analysis. But I can move it in.

Reply:
If we need for something else down the road then we can move it. I'm starting to lean towards coding for now and not something that doesn't exist quite yet, but can be convinced otherwise

Reply:
I should have elaborated more. In the issue we concluded to not move it out for this same reason. At the time I had moved it into the utils as I was thinking of moving it to spatialnc since we decided not to I just left it. But as you suggested the correct conclusion is to move all the way into the topo class. I agree, preferring current code over ephemeral-might-be code is a good strategy. lol